### PR TITLE
Enable other GHCs with Parsec (for Backpack coverage.)

### PR DESCRIPTION
**Pull request by ezyang**
Signed-off-by: Edward Z. Yang ezyang@cs.stanford.edu

### CODE REVIEW

**Comment by ezyang** [comment]
Merged.

**Comment by phadej** [comment]
Whst happened to Travis file?

**Comment by ezyang** [comment]
Uhh, not much? https://github.com/haskell/cabal/commit/20bed0639ac7dd50149a98f5dd1a0cda09f9e5b4

File: .travis.yml [hunk 1]
```diff
@@ -19,65 +19,9 @@ branches:
 # lines listings versions you don't need/want testing for.
 matrix:
   include:
-   - env: GHCVER=8.0.1 SCRIPT=meta BUILDER=none
```
Comment (phadej) [inline .travis.yml]: @ezyang yo merged this commit.

**Comment by phadej** [comment]
Ah, you pushed directly to master. Sorry. It's just very confusing. I look at PR see, it's merged, but its changes are *wrong*. 

I think we should *protect* master branch and disallow direct pushes. There is still way to admin's to go around that limitation, but at least you'll stop and think.

Especially this all was confusing as mention-bot pinged me, and no-one said I don't need to do anything.

**Comment by ezyang** [comment]
Sorry, I'll make sure the PR is up-to-date before merging.

The irritating thing about our Travis setup is it's too dumb to know that I made a trivial change that doesn't actually need to be rebuilt; just a waste of queue (and impedes other people who are more interested in their builds.) So a direct push is appealing because it avoids the useless Travis build. Would be far better if we could do something else, like how homu works.

**Pull request closed without merging**